drivers/soft_spi/include/soft_spi_params.h

```diff
@@ -42,12 +42,14 @@ extern "C" {
 #endif
 
 /**
- * @brief   Sotware SPI port descriptor array
+ * @brief   Software SPI port descriptor array
  */
 static soft_spi_conf_t soft_spi_config[] = {
     SOFT_SPI_PARAMS,
 };
 
+#define SOFT_SPI_NUMOF             ARRAY_SIZE(soft_spi_config)
```

Comment:
```suggestion
/**
 * @brief   Number of software SPI buses
 */
#define SOFT_SPI_NUMOF             ARRAY_SIZE(soft_spi_config)
```

This should get the doc check happy

Reply:
done